plugins/testplugin/src/testplugin.cpp

```diff
@@ -332,7 +332,7 @@ void TestPlugin::initMetadata()
 	{
 	   "priority":2,
 	   "category":[
-	      "test"
+	      "test", "iio"
```

Comment:
Test plugin shouldn't have iio category in "production". Using it locally is totally fine.